### PR TITLE
Introduce content from kubernetes-template-project

### PR DESCRIPTION
**Pull request by eparis**
This did not include a README or a RELEASE file.

### CODE REVIEW

**Comment by eparis** [comment]
Partially solves #1

**Comment by pmorie** [comment]
@eparis - rebase needed

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Follow either of the two links above to access the appropriate CLA and instructions for how to sign and return it. Once we receive it, we'll be able to accept your pull requests.
+
+### Contributing A Patch
+
+1. Submit an issue describing your proposed change to the repo in question.
+1. The [repo owners](OWNERS) will respond to your issue promptly.
+1. If your proposed change is accepted, and you haven't already done so, sign a Contributor License Agreement (see details above).
+1. Fork the desired repo, develop and test your code changes.
+1. Submit a pull request.
```
Comment (munnerz) [inline CONTRIBUTING.md]: It seems other staged repos don't have this file to use as a reference, but the contributing guidelines here do not apply. Patches need to be contributed upstream in k/k.

Comment (munnerz) [reply CONTRIBUTING.md]: Perhaps we should have a separate project template for these staged repos, and some way of enforcing that repos follow this template too 🤔

**Comment by nikhita** [comment]
It would be better to include this in https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/code-generator since README, etc exist there.

**Comment by sttts** [comment]
We have those files now.

**Pull request closed without merging**